### ISSUE TITLE
E2E testing of revisions

### ISSUE THREAD

**Issue by MDunitz** [chanzuckerberg/single-cell-data-portal#1286]
## Need
Confidence that the revision feature works as expected 

## Approach
- FE moved to a [separate ticket](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/1292)

- Backend testing via functional tests of APIs in staging
     - [ ] Updating a dataset in a revision does not effect the published dataset
     - [ ] Publishing a revised dataset replaces the original dataset
        - Check the metadata api to ensure the s3_uri has changed
       - Check the the explorer/schema endpoint to ensure the explorer is accessing the correct dataset
    - [ ] Deleting a dataset does not effect the published dataset
    - [ ] Publishing a revision that deletes a dataset removes it from the data portal (And the explorer url redirects appropriately)
  - [ ] Adding a dataset to a revision does not impact public datasets in that collection
  - [ ] Publishing a revision that contains a new dataset updates the collections page for the data portal (with the new dataset)

**Comment by maniarathi** [comment]
Potentially split into two. @seve , @MDunitz

**Comment by brianraymor** [comment]
I added a few cases to the work items above.

Are you confident about the UX for updating datasets and adding datasets on the FE - labels for example? And variations like add-then delete?

**Comment by tihuan** [comment]
blocked by BE ticket #2234
